src/olympia/lib/akismet/models.py

```diff
@@ -82,27 +90,51 @@ def comment_check(self):
                 self.update(result=(
                     self.HAM if not outcome else
                     self.DEFINITE_SPAM if discard else self.MAYBE_SPAM))
+                log.debug('Akismet response %s' % self.get_result_display())
                 return self.result
         except ValueError:
             # if outcome isn't valid json `response.json` will raise ValueError
             pass
         log.error(
-            'Akismet error %s' % (
+            'Akismet comment-check error: %s' % (
                 response.headers.get('X-akismet-debug-help') or response.text))
         self.update(result=self.UNKNOWN)
         return self.result
 
+    def _submit(self, spam_or_ham):
+        response = self._post('submit-%s' % spam_or_ham)
```

Comment:
Let's maybe be paranoid and add an `assert spam_or_ham in ('ham', 'spam')` just to make this interface more explicit